draft-ietf-httpbis-digest-headers.md

```diff
@@ -354,16 +355,11 @@ the `Content-Digest` field.
    Want-Digest = 1#want-digest-value
    Want-Content-Digest = 1#want-digest-value
    want-digest-value = digest-algorithm [ ";" "q" "=" qvalue]
-   qvalue = ( "0"  [ "."  0*1DIGIT ] ) /
-            ( "1"  [ "."  0*1( "0" ) ] )
 ~~~
 
-qvalue (see Section 12.4.2 of {{SEMANTICS}}) indicates the sender's preference
-from highest (1.0) to lowest (0.0). If a digest-algorithm is not accompanied by a
-"qvalue" , it is treated as if its associated "qvalue" were 1.0.
+`qvalue` indicates the sender's digest-algorithm's preferences.
```

Comment:
```suggestion
`qvalue` indicates the sender's digest-algorithm preferences.
```
They are the sender's preferences about algorithms, not the preferences of the algorithms themselves.

Reply:
ahh such a bad use of an `'s` by me, the shame